docs/guide/migration.md

```diff
@@ -261,14 +261,14 @@ export default [
     ],
   }),
   // override the legacy rules
-  StylisticPlugin.configs['disable-legacy'] // [!code ++],
+  StylisticPlugin.configs['disable-legacy'] // [!code ++]
   // your own rules
   {
     plugins: {
       stylistic: StylisticPlugin
     },
     rules: {
-      'stylistic/semi': 'error',
+      '@stylistic/semi': 'error',
```

Comment:
No, this is not how it works. The previous example is correct as we provide the plugin named as `stylistic`

Reply:
`@stylistic` was mainly the convention of the legacy config, where in the flat config I don't want users to carry that legacy convention when they don't need to.